### PR TITLE
check djob is submitted before creating run to prevent crashing

### PR DESCRIPTION
**Pull request by joshagudo**
Addresses #136

### CODE REVIEW

File: controllers/run_controller_databricks.go [hunk 1]
```diff
@@ -64,6 +64,10 @@ func (r *RunReconciler) submit(instance *databricksv1alpha1.Run) error {
 			},
 		})
 
+		if !k8sJob.IsSubmitted() {
+			return fmt.Errorf("Run references Djob that is not yet submitted")
```
Comment (Azadehkhojandi) [inline controllers/run_controller_databricks.go]: Do you need to set the Requeue to True?

Comment (joshagudo) [reply controllers/run_controller_databricks.go]: Yes you're right - I've added that in.

File: controllers/run_controller_test.go [hunk 2]
```diff
@@ -145,4 +145,76 @@ var _ = Describe("Run Controller", func() {
...
+			jobKey := types.NamespacedName{
+				Name:      "t-job-for-run" + "-" + randomStringWithCharset(10, charset),
+				Namespace: "default",
+			}
+
+			jobSpec := &databricksv1alpha1.JobSettings{
+				NewCluster: &dbmodels.NewCluster{
+					SparkVersion: "5.3.x-scala2.11",
+					NodeTypeID:   "Standard_D3_v2",
+					NumWorkers:   10,
```
Comment (Azadehkhojandi) [inline controllers/run_controller_test.go]: Can you please change it to 3?

**Pull request merged**